aikido_perception/include/aikido/perception/yaml_conversion.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef AIKIDO_PERCEPTION_YAMLCONVERSION_H
+#define AIKIDO_PERCEPTION_YAMLCONVERSION_H
+
+#include <yaml-cpp/yaml.h>
+#include <Eigen/Dense>
+
+namespace YAML {
+
+template < typename _Scalar, int _Rows, int _Cols, int _Options, int _MaxRows, int _MaxCols>
+    struct convert< Eigen::Matrix<_Scalar, _Rows, _Cols, _Options, _MaxRows, _MaxCols> > {
+        static Node encode(const Eigen::Matrix<_Scalar, _Rows, _Cols, _Options, _MaxRows, _MaxCols>& matrix) {
+            Node node(NodeType::Sequence);
+
+            int nValues = matrix.rows() * matrix.cols();
+
+            for (int i = 0; i < nValues; ++i) {
+                node.push_back( matrix(i / matrix.cols(), i % matrix.cols() ) );
+            }
```

Comment:
Done the change after removing the `Tag()` stuff from the `muul` `yaml_utils` file